.github/workflows/scheduled-repository-template-sync.yaml

```diff
@@ -0,0 +1,55 @@
+name: Repository Template Sync
+
+on:
+  schedule:
+    - cron: '0 0 1 * *'
+  workflow_dispatch:
+
+permissions:
+  contents: write
+  pull-requests: write
+  checks: read
+
+jobs:
+  update-external-repo:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out the repository
+      uses: actions/checkout@v3
+
+    - name: Check out external repository
+      uses: actions/checkout@v3
+      with:
+        repository: nhsdigital/nhs-notify-repository-template
+        path: nhs-notify-repository-template
+
+    - name: Run syncronisation script
+      run: |
+        /scripts/githooks/sync-template-repo.sh
+
+    - name: Create Pull Request
```

Comment:
What happens if there is no drift?

Reply:
according to the behaviours on the action

> If there are no changes (i.e. no diff exists with the checked-out base branch), no pull request will be created and the action exits silently.

https://github.com/peter-evans/create-pull-request/tree/v7.0.1/?tab=readme-ov-file#action-behaviour